libs/core/src/lib/carousel/carousel.component.html

```diff
@@ -66,17 +66,15 @@
 </ng-template>
 
 <ng-template #buttonLeft>
-    <button class="fd-button fd-carousel__button fd-carousel__button--left" [style.z-index]="1" data-slide="previous"
-            [attr.aria-label]="leftNavigationBtnLabel" (click)="previous(true); $event.stopPropagation();"
-            [disabled]="leftButtonDisabled" [attr.title]="leftNavigationBtnLabel">
-        <i [class]="_isRtl() ? 'sap-icon--slim-arrow-right' : 'sap-icon--slim-arrow-left'"></i>
-    </button>
+    <button fd-button class="fd-carousel__button fd-carousel__button--left" [style.z-index]="1" data-slide="previous"
+        [attr.aria-label]="leftNavigationBtnLabel" (click)="previous(true); $event.stopPropagation();"
```

Comment:
Does the flag usage needed here? Same for the next method. Seems like it's always used with value `true`

Reply:
after refactor, it is not needed. corrected the logic here, for the focus after last item is reached.
fixed now.